packages/react/src/hooks/useInputState/useInputState.spec.tsx

```diff
@@ -0,0 +1,157 @@
+import { describe } from 'node:test';
+import { useInputState } from '.';
+import { expect, it } from 'vitest';
+import { renderHook, screen } from '@testing-library/react';
+import { renderSetup } from '../../_internal/test/renderSetup';
+import { userEvent } from '@testing-library/user-event';
+
+const TestComponent = ({
+  initial,
+  isObject,
+  isName,
+}: {
+  initial: any;
+  isObject: boolean;
+  isName: boolean;
+}) => {
+  const { value, onChange, reset } = useInputState(initial);
+
+  return (
+    <>
+      <h1 role="test-value">{isObject ? value.test : value}</h1>
+      {isName ? (
+        <input
+          data-testid="test-input"
+          name="test"
+          onChange={onChange}
+          value={isObject ? value.test : value}
+        />
+      ) : (
+        <input
+          data-testid="test-input"
+          onChange={onChange}
+          value={isObject ? value.test : value}
+        />
+      )}
+      <button onClick={reset}>reset</button>
+    </>
+  );
+};
+
+describe('useInputState', () => {
+  it('value값이 단일 값일 때, name이 없어도 값이 변경됩니다.', async () => {
+    renderSetup(
+      <TestComponent
+        initial={'initial value'}
+        isObject={false}
+        isName={false}
+      />
+    );
```

Comment:
```suggestion
    const { user } = renderSetup(
      <TestComponent
        initial={'initial value'}
        isObject={false}
        isName={false}
      />
    );
    
    // ...
    
    await user.type(input, ' changed');
```
renderSetup 에서 user를 반환합니다. 이를 활용해주세요!
상단에 userEvent import도 제거해주시면 감사드립니다

Reply:
@ssi02014 수정했습니다!